devops/azure-pipelines/azure-pipelines.yml

```diff
@@ -13,26 +11,29 @@ pool:
 stages:
   - stage: Build_and_Test
     jobs:
-      - template: ./gradle-build-test-job-template.yml
+      - template: ./maven-build-test-job-template.yml
         parameters:
           jobName: 'Build_and_Test'
-          testPath: '**/TEST-*.xml'
-          ccPath: 'build/classes/main/'
+          serviceConnection: 'Forge-SonarCloud'
+          organization: '1058-sophia-gavrila-forge'
+          testPath: '**/surefire-reports/TEST-*.xml'
   - stage: Dockerize
     jobs:
       - template: ./dockerize-template.yml
         parameters:
           jobName: 'Dockerize'
           containerRegistry: 'project-3-docker-hub'
           repository: 'dajpearl/forge-backend'
-          imageTag: '${{ variables.Build.SourceVersion }}'
-          dockerPath: '**/gradle.Dockerfile'
+          imageTag: "$(Build.SourceVersion)"
+          dockerPath: '**/mavenpipe.dockerfile'
   - stage: Deploy
     jobs:
       - template: ./kubernetes-template.yml
         parameters:
           jobName: 'Deploy'
-          serviceConnection: 'may24-devops-team2-dev-cluster'
+          branchMain: 'refs/heads/azure-pipelines'
+          serviceConnectionMain: 'Alex-Main-Cluster'
+          serviceConnectionDev: 'Alex-Dev-Cluster'
```

Comment:
why are we putting alex's cluster on dev branch?

Reply:
We are not putting my cluster on the dev branch, we wanted a review of the changes we have made thus far, but fully intend on putting the right service connections back in once they are approved in the Terraform groups pr. We have no plans on merging before these changes are made, we just wanted a review of everything we have worked on on this branch up to this point.

Reply:
ok